cmd/json.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/sourcegraph/sourcegraph/lib/errors"
+	"github.com/urfave/cli/v2"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+type jsonFlags struct {
+	from   string
+	pretty bool
+}
+
+func jsonCommand() cli.Command {
+	var jsonFlags jsonFlags
+
+	json := cli.Command{
+		Name:  "json",
+		Usage: "Print SCIP index as JSON",
+		Flags: []cli.Flag{
+			fromFlag(&jsonFlags.from),
```

Comment:
If anyone knows how to make this argument positional, I'd appreciate it
Just using the NArgs leads to the flags being ignored

`scip json -p --from index.scip` is not terrible, but leaves a lot to be desired